ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -713,6 +765,10 @@ public void getNotModifiedBlobAndVerify(String blobId)
         restResponseChannel.getHeader(RestUtils.Headers.BLOB_SIZE));
     assertNull("Content-Type should have been null", restResponseChannel.getHeader(RestUtils.Headers.CONTENT_TYPE));
     assertEquals("No content expected as blob is not modified", 0, restResponseChannel.getResponseBody().length);
+    assertEquals("Accept-Ranges not set correctly", "bytes",
```

Comment:
why would a `NOT_MODIFIED` response have a `ACCEPT_RANGES` header? Is that what is expected?